### ISSUE TITLE
error when saving a row

### ISSUE THREAD

**Issue by cybrwshl** [nocodb/nocodb#412]
NocoDB used as docker : true
NocoDB version : 0.11.6
Database used in NC_DB URL : pg
Project was created by clicking : New Project
Database on which spreadsheet is created : pg
OS on which NocoDB is running : linux
Node.js version if running as node :
Database version : 12.1

**Steps To Reproduce**
1. create new row
2. click on row form and enter some data
3. click 'Save Row'
4. error message pops up
5. row is saved

**Expected behavior**
Success message

**Screenshots**
`Error: Promised response from onMessage listener went out of scope ExtensionMessagingService.js:89:34`
![grafik](https://user-images.githubusercontent.com/5388891/127842292-7a4089cc-1769-4213-a5d6-dcf99c19c3e8.png)

Node: **v12.22.1**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **pg**
ProjectOnRootDB: **true**
RootDB: **pg**
PackageVersion: **0.11.6**

**Comment by pranavxc** [comment]
Is there any other error log in server or browser console?

**Comment by cybrwshl** [comment]
in the firefox console i saw this message:
```
Error: Promised response from onMessage listener went out of scope ExtensionMessagingService.js:89:34
```
Node didn't report any errors. Only message was the select used to retrieve the newly added dataset.

**Comment by pranavxc** [comment]
> in the firefox console i saw this message:
> 
> ```
> Error: Promised response from onMessage listener went out of scope ExtensionMessagingService.js:89:34
> ```
> 
> Node didn't report any errors. Only message was the select used to retrieve the newly added dataset.

Does the data is getting inserted?

**Comment by cybrwshl** [comment]
yep the data is getting inserted

**Comment by pranavxc** [comment]
> yep the data is getting inserted

OK, you are trying to insert it using expanded form right?

**Comment by cybrwshl** [comment]
yep. I've used the form.

**Comment by pranavxc** [comment]
> yep. I've used the form.

Thanks for confirming. One last question does it causing the same problem now as well?

**Comment by cybrwshl** [comment]
what do you mean by now?

**Comment by pranavxc** [comment]
> what do you mean by now?

Currently whenever you insert a new record does the error message is showing? Is the problem still persist or it's just happened one-time?

**Comment by pranavxc** [comment]
It's happened due to a missed argument and the issue is already fixed in the latest version (`0.11.4`)
